demo/scripts/controls/sidePane/eventViewer/EventViewPane.tsx

```diff
@@ -62,6 +62,7 @@ const EntityOperationMap: { [key in EntityOperation]: string } = {
     [EntityOperation.RemoveFromStart]: 'RemoveFromStart',
     [EntityOperation.ReplaceTemporaryContent]: 'ReplaceTemporaryContent',
     [EntityOperation.UpdateEntityState]: 'UpdateEntityState',
+    [EntityOperation.SnapshotEntityState]: 'SnapshotEntityState',
```

Comment:
As I said, no need to add this new event type for old editor. This should be a new feature for new editor only. Old editor should never receive this event.

Reply:
If I do not include it, I get this error:

```
[tsl] ERROR in C:\Users\fjanke\Documents\code\roosterjs\demo\scripts\controls\sidePane\eventViewer\EventViewPane.tsx(52,7)
      TS2741: Property '12' is missing in type '{ 9: string; 10: string; 1: string; 2: string; 3: string; 0: string; 6: string; 7: string; 5: string; 4: string; 8: string; 11: string; }' but required in type '{ 0: string; 1: string; 2: string; 3: string; 4: string; 5: string; 6: string; 7: string; 8: string; 9: string; 10: string; 11: string; 12: string; }'.
```

because `EntityOperation` has `SnapshotEntityState` I need a matching mapping in `EntityOperationMap`